libraries/apollo-tooling/src/main/kotlin/com/apollographql/apollo3/tooling/SchemaHelper.kt

```diff
@@ -38,7 +36,29 @@ internal object SchemaHelper {
     return clientBuilder.build()
   }
 
-  internal fun executeQuery(map: Map<String, Any?>, url: String, headers: Map<String, String>, insecure: Boolean): Response {
+  internal fun executeQuery(
+      query: Query<*>,
+      endpoint: String,
+      headers: Map<String, String>,
+      insecure: Boolean,
+  ): String {
+    val composer = DefaultHttpRequestComposer(endpoint)
+    val apolloRequest = ApolloRequest.Builder(query)
+        .httpHeaders(headers.map { HttpHeader(it.key, it.value) })
+        .build()
+    val httpRequest = composer.compose(apolloRequest)
+    val httpEngine = DefaultHttpEngine(newOkHttpClient(insecure))
+    val httpResponse = runBlocking { httpEngine.execute(httpRequest) }
+    val bodyStr = httpResponse.body?.use {
+      it.readUtf8()
+    }
```

Comment:
```suggestion
    val bodyStr = httpResponse.body?.string()
```

Reply:
Can't find this one, is it an extension in Apollo?

Reply:
Nevermind, I though we were in OkHttp land 👍 (although we might put it in ApolloClient 👀 )